src/components/NavBar/MobileNavItems.tsx

```diff
@@ -17,7 +17,14 @@ import { NavItemsProps, navItems } from './NavItems';
  * The buttons are currently hardcoded but as these are unlikely to change, hopefully
  * this isn't an issue.
  */
-function MobileNavItems({ classNameValue, isLoggedIn }: NavItemsProps) {
+
+interface MobileNavItemsProps {
+	classNameValue: string;
+	isLoggedIn: boolean; 
+  }
+
+
+function MobileNavItems({ classNameValue, isLoggedIn }: MobileNavItemsProps) {
```

Comment:
Can you explain why you changed this? NavItemsProps has exactly the same props which is why I imported and used it instead of reproducing it...

Reply:
Done, thanks